nautobot/extras/forms/forms.py

```diff
@@ -1143,7 +1143,7 @@ class ObjectChangeFilterForm(BootstrapMixin, forms.Form):
 
 
 class RelationshipForm(BootstrapMixin, forms.ModelForm):
-    slug = SlugField(help_text="Internal name of this relationship. Please use underscores rather than dashes.")
+    key = SlugField(help_text="Internal name of this relationship. Please use underscores rather than dashes.")
```

Comment:
What do we need to do UI-side to switch the default slugification for this field from dashes to underscores? We did that with custom fields already right?

Reply:
Yeah it is already addressed in the CustomField PR here. https://github.com/nautobot/nautobot/pull/3426/files#diff-8906c95050cdc47ee55226257d68c10271005536e061de29fc6a256d4bbfe102